client/my-sites/upgrades/checkout/supporting-text.jsx

```diff
@@ -72,17 +71,7 @@ var SupportingText = React.createClass( {
 	},
 
 	render: function() {
-		if ( abtest( 'checkoutFooter' ) === 'noFooter' ) {
 			return null;
```

Comment:
This file probably needs to be removed completely. cc: @scruffian for some help / guidance here.